backend/build.gradle.kts

```diff
@@ -59,6 +59,10 @@ tasks.register("setupEnvironmentVariables") {
         if (System.getenv("CRD_DB_PASSWORD") == null) {
             throw IllegalStateException("Required environment variable is not defined: CRD_DB_PASSWORD")
         }
+
+        if (System.getenv("FILE_UPLOAD_DIRECTORY") == null) {
```

Comment:
is this a variable we need to do in setup? If so, can you update the readme accordingly?

Reply:
I just added a default